src/main/java/org/fundacionjala/coding/abel/SortInnerContent.java

```diff
@@ -0,0 +1,38 @@
+package org.fundacionjala.coding.abel;
+
+
+import java.util.Collections;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+/**
+ * Created by AbelBarrientos on 5/23/2017.
+ */
+public class SortInnerContent {
+
+    /**
+     * This method is used to sort the inner content of a word.
+     * @param word This is the word parameter to sort.
+     * @return String This return the word sorted.
+     */
+    public String sort(String word) {
+        StringBuilder stringBuilder = new StringBuilder();
+        stringBuilder.append(word.charAt(0));
+        Stream.of(word.substring(1, word.length() - 1).split(""))
+                .sorted(Collections.reverseOrder())
+                .forEach(character -> stringBuilder.append(character));
+        stringBuilder.append(word.charAt(word.length() - 1));
```

Comment:
You can try this to avoid the forEach.
```java
String innerDescSorted = Stream.of(word.substring(1, word.length() - 1).split(""))
                                .sorted(Comparator.reverseOrder())
                                .collect(Collectors.joining());
stringBuilder.append(character);
```
if you apply that, it will not be necessary to have a StringBuilder.